docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Graph-Backed Assets | Dagster
+description: Defining a software-defined asset with multiple discrete computations combined in a graph.
+---
+
+# Graph-Backed Assets
+
+A graph-backed asset is a software-defined asset whose computations are separated into multiple [ops](/concepts/ops-jobs-graphs/ops) that are combined to build a [graph](/concepts/ops-jobs-graphs/graphs). A graph-backed asset can output multiple assets.
+
+---
+
+## Relevant APIs
+
+| Name                                                       | Description                                   |
+| ---------------------------------------------------------- | --------------------------------------------- |
+| <PyObject object="AssetsDefinition" method="from_graph" /> | Constructs an asset given a graph definition. |
+
+---
+
+## Overview
```

Comment:
As I've been going through the Concept pages, I've been consolidating what was the **Overview** into the actual introduction section of the page. This eliminates what was a redundant/oddly-placed heading and makes the intro an actual introduction. Could we please do that here?

Reply:
updated!